### ISSUE TITLE
[0.9.1] Unable to create series once it is dropped

### ISSUE THREAD

**Issue by buro1983** [influxdb/influxdb#3226]
I am using influxdb-0.9.1-1.x86_64. 

Scenario:
======
1. Created a series using curl
2. Executed "drop series from <series name>"
3. Then again tried to create series with same name using curl. Executed show series from CLI but did get any output.

Is it expected behavior?

**Comment by otoolep** [comment]
Can you please show the entire sequence using the actual `curl` commands? That would be most helpful.

**Comment by buro1983** [comment]
curl -i -XPOST 'http://ip:port/write?db=mydb&precision=s' --data-binary Usage,applicationtype=WebBrowsing,application=google Upstream=10000,Downstream=200000 1435917290'

**Comment by otoolep** [comment]
What I mean @buro1983 is the entire sequence -- creating the database, writing a point, demonstrating the data is there, dropping the series, and then re-attempting the write. Please include the output of each command.

https://github.com/influxdb/influxdb/blob/master/CONTRIBUTING.md

**Comment by samhatchett** [comment]
i'm experiencing the same problem as described above. to reproduce:

````
# create database
curl -G http://localhost:8086/query --data-urlencode "q=CREATE DATABASE drop_bug"

# send a point
curl -i -XPOST 'http://localhost:8086/write?db=drop_bug' --data-binary 'my_metric,host=myhost value=0.64 1434055562000000000'

# prove that the series is there.
curl -G http://localhost:8086/query  --data-urlencode 'db=drop_bug' --data-urlencode 'q=SHOW SERIES'

# DROP
curl -G http://localhost:8086/query  --data-urlencode 'db=drop_bug' --data-urlencode 'q=DROP SERIES FROM my_metric'

# send a point - yes, the same point.
curl -i -XPOST 'http://localhost:8086/write?db=drop_bug' --data-binary 'my_metric,host=myhost value=0.64 1434055562000000000'

# Try to show series again. It should show the series just re-created, but comes back empty.
curl -G http://localhost:8086/query  --data-urlencode 'db=drop_bug' --data-urlencode 'q=SHOW SERIES'
````


my system: Mac OSX 10.10.4, InfluxDB 0.9.1 installed via Brew

**Comment by samhatchett** [comment]
so is there currently any way to delete data from a db without restarting afterwards? I see from #1647 that DELETE is not yet implemented (`ERR: invalid query`) - so how are we to accomplish deletions?

**Comment by beckettsean** [comment]
@samhatchett this bug relates to being unable to insert new points into the dropped series. The data in the series is dropped successfully.

**Comment by ninokop** [comment]
[0.9.1] I'm experiencing the same problem.

> select * from cpu
name: cpu
tags: host=local
time				value
----				-----
2015-07-09T16:54:40.740627311Z	0.6
> drop series from cpu where host='local'
> select * from cpu
> insert cpu,host=local value=9.0
> select * from cpu
> 
> 
> show series
>

It seems DROP does not succeed. When restarting the influxd process and writing data, the series that I dropped appear again.

> select * from cpu
> insert cpu,host=local value=7.0
> select * from cpu
name: cpu
tags: host=local
time				value
----				-----
2015-07-09T16:54:40.740627311Z	0.6
2015-07-09T16:55:27.193514255Z	9
2015-07-09T16:56:54.869625619Z	7
...

**Comment by samhatchett** [comment]
@beckettsean I get it - what I should have said is that obviously this is not currently a preferred method of deletion since it renders a time series defunct for future writes. Therefore it's off the list of things I can use to delete data.

**Comment by beckettsean** [comment]
@samhatchett that is correct, it's not currently a good way to delete. It is supposed to work, though, so starting with 0.9.2 or at worst 0.9.3 this should be the correct way to do what you're trying to accomplish.

**Comment by beckettsean** [comment]
Additional details on this problem in #3283

**Comment by otoolep** [comment]
Fixed by https://github.com/influxdb/influxdb/pull/3651. Thanks for the reports @buro1983 and @samhatchett.

**Comment by otoolep** [comment]
Fix will be in nightlies and 0.9.3.
